web/tests/peer_data.test.cjs

```diff
@@ -73,6 +76,11 @@ function test(label, f) {
         user_groups.initialize({
             realm_user_groups: [nobody_group],
         });
+        override(
+            realm,
+            "server_supported_permission_settings",
+            example_settings.server_supported_permission_settings,
+        );
```

Comment:
This can be a follow-up after 10.0, but probably we to replace this override with just initializing `server_supported_permission_settings` properly in the node test setup code.